### PR TITLE
Seed City and Images

### PR DESCRIPTION
**Pull request by ThanhHoai511**
Seed City and Images

### CODE REVIEW

**Comment by ThanhHoai511** [comment]
ready

File: database/seeds/CitiesTableSeeder.php [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        $hcm = [
+        	'Quận 1',
+			'Quận 12',
```
Comment (BunnyPi04) [inline database/seeds/CitiesTableSeeder.php]: tab lại

File: database/seeds/ImagesTableSeeder.php [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+    	$faker = Faker\Factory::create();
+        for($i = 1; $i<= 23; $i++)
```
Comment (BunnyPi04) [inline database/seeds/ImagesTableSeeder.php]: convention

File: storage/.gitignore [hunk 3]
```diff
@@ -0,0 +1,2 @@
+*
+!.gitignore
```
Comment (BunnyPi04) [inline storage/.gitignore]: thiếu dòng trắng cuối file

**Comment by BunnyPi04** [comment]
commented

**Comment by ThanhHoai511** [comment]
ready

**Pull request merged**